app/src/main/java/com/example/workoutapp/ProfileActivity.kt

```diff
@@ -171,7 +173,8 @@ fun UserProfile(
                         text = "Goals",
                         style = MaterialTheme.typography.h5,
                         fontWeight = FontWeight.Bold,
-                        modifier = Modifier.padding(bottom = 8.dp)
+                        modifier = Modifier.padding(bottom = 8.dp),
+                        textAlign = TextAlign.Center
```

Comment:
UI changes are design decisions.  They don't impact technical debt, unless it increases accessibility in some (documented) manner, like changing color contrast.